### PR TITLE
Support JS/TS file format

### PR DESCRIPTION
**Pull request by jeeyi**
WIP - do not merge yet.

This is to support JS file format. For example,
```
export default {
  // Comment 1
  "be.cancel": "Cancel",
  // Comment 2
  "be.cancelUploads": "Cancel Uploads",
. . .
}
```

### CODE REVIEW

File: webapp/src/test/java/com/box/l10n/mojito/service/assetExtraction/extractor/AssetPathToFilterConfigMapperTest.java [hunk 1]
```diff
@@ -105,4 +105,20 @@ public void testGetFilterConfigIdFromTypeWithPo() throws Exception {
...
+        AssetPathToFilterConfigMapper assetPathToFilterConfigMapper = new AssetPathToFilterConfigMapper();
+        String filterConfigId = assetPathToFilterConfigMapper.getFilterConfigIdFromPath("/path/to/File.ts");
+        assertEquals(AssetPathToFilterConfigMapper.JS_FILTER_CONFIG_ID, filterConfigId);
+    }
+
+    @Test
+    public void testGetFilterConfigIdFromTypeWithJS() throws Exception {
+
+        AssetPathToFilterConfigMapper assetPathToFilterConfigMapper = new AssetPathToFilterConfigMapper();
+        String filterConfigId = assetPathToFilterConfigMapper.getFilterConfigIdFromPath("/path/to/File.js");
```
Comment (ehoogerbeets) [inline webapp/src/test/java/com/box/l10n/mojito/service/assetExtraction/extractor/AssetPathToFilterConfigMapperTest.java]: Also jsx files?

File: webapp/src/main/java/com/box/l10n/mojito/service/assetExtraction/extractor/AssetPathToFilterConfigMapper.java [hunk 2]
```diff
@@ -46,7 +48,8 @@
         MACSTRINGS(MACSTRINGS_FILTER_CONFIG_ID, "strings"),
         PO(POFilter.FILTER_CONFIG_ID, "pot"),
         RESX(RESX_FILTER_CONFIG_ID, "resx", "resw"),
-        XTB(XTB_FILTER_CONFIG_ID, "xtb");
+        XTB(XTB_FILTER_CONFIG_ID, "xtb"),
+        JS(JS_FILTER_CONFIG_ID, "ts", "js");
```
Comment (ehoogerbeets) [inline webapp/src/main/java/com/box/l10n/mojito/service/assetExtraction/extractor/AssetPathToFilterConfigMapper.java]: and jsx files too?

Comment (jeeyi) [reply webapp/src/main/java/com/box/l10n/mojito/service/assetExtraction/extractor/AssetPathToFilterConfigMapper.java]: Do you know any teams using jsx file for string resources?

**Comment by jeeyi** [comment]
@aurambaj Yes, string id and value are in `"` and in one line.

**Pull request merged**